api/src/main/java/javax/observer/collection/CollectionObservable.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package javax.observer.collection;
+
+import java.util.Collection;
+import javax.observer.Subscription;
+
+/**
+ * 
+ * @param <T> element type of the collection
+ * @param <OC> observed collection type
+ * @param <CC> collection change type
+ * 
+ * @author Michael Hoffer (info@michaelhoffer.de)
+ */
+public interface CollectionObservable<T, OC extends Collection<T>, CC extends CollectionChange<T>> {
```

Comment:
I don't like ObservableCollection and the like. The ide is that one can have the use case of observables that track changes of collections but are no collections themselves. But I'm fine with another name that expresses the same meaning.